### PR TITLE
Add the version code to the Quest android build

### PR DESCRIPTION
**Pull request by jherico**
[21591](https://highfidelity.manuscript.com/f/cases/21591/Add-version-code-to-Quest-interface-builds-so-that-they-can-be-pushed-to-the-Oculus-store)

### CODE REVIEW

File: android/apps/interface/build.gradle [hunk 1]
```diff
@@ -36,11 +36,6 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 26
-    //buildToolsVersion '27.0.3'
-
-    def appVersionCode = Integer.valueOf(VERSION_CODE ?: 1)
```
Comment (jherico) [inline android/apps/interface/build.gradle]: these are already defined in the parent gradle

File: android/apps/questInterface/build.gradle [hunk 2]
```diff
@@ -18,12 +18,11 @@ task renameHifiACTaskRelease(type: Copy) {
 
 android {
     compileSdkVersion 28
-
     defaultConfig {
         applicationId "io.highfidelity.questInterface"
         minSdkVersion 24
         targetSdkVersion 28
-        versionCode 1
+        versionCode appVersionCode
```
Comment (jherico) [inline android/apps/questInterface/build.gradle]: allow updated PR builds to be sent to the Oculus store.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17910/HighFidelity-Beta-PR15112-d4cea60.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17910/HighFidelity-Quest-Beta-PR15112-d4cea60.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17910/HighFidelity-Beta-PR15112-d4cea60.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17910/HighFidelity-Beta-Interface-PR15112-d4cea60.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17910/HighFidelity-Beta-PR15112-d4cea60.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17910/HighFidelity-Beta-Interface-PR15112-d4cea60.exe

**Pull request merged**